sphinx_antsibull_ext/pygments_lexer.py

```diff
@@ -169,7 +169,7 @@ class AnsibleOutputLexer(DelegatingLexer):
     '''
 
     name = 'Ansible-output'  # pyre-ignore[15]
-    aliases = ['ansible-output']
+    aliases = ('ansible-output', )
```

Comment:
Is this significant? Anything requires tuples?

Reply:
The `LEXERS` structure did require tuples (and pyre had some trouble with locally converting this to a tuple). Since that code is gone, there's no reason to keep this as a tuple (also the lexers in Pygments itself use lists).